### PR TITLE
Add a link to juju docs pages on the integration tab 

### PR DESCRIPTION
**Pull request by chillkang**
## Done
- Add a link to juju docs pages next to a heading on the integration tab 

## How to QA
- Go to https://charmhub-io-1787.demos.haus/mongodb/integrations
- Check if you see `Learn about integrations >` link which directs to `https://juju.is/docs/juju/relation`

## Issue / Card
Fixes #https://warthogs.atlassian.net/browse/WD-9926

## Screenshots
Screenshot 2024-04-09 at 2 06 12 PM

### CODE REVIEW

File: static/js/src/public/details/integrations/components/App/App.tsx [hunk 1]
```diff
@@ -150,10 +150,15 @@ export const App = () => {
           <Col size={9} className="p-details-tab__content__body">
             <Row>
               <Col size={5}>
-                <h2 className="p-heading--3">
+                <h2 className="p-heading--3 p-details-tab__content__body__title">
                   {integrationCount} integration
-                  {integrationCount > 1 ? "s" : ""} related to this charm
+                  {integrationCount > 1 ? "s" : ""}
                 </h2>
+                <h3 className="p-heading--4 p-details-tab__content__body__link">
```
Comment (steverydz) [inline static/js/src/public/details/integrations/components/App/App.tsx:157]: This shouldn't be a heading. A `p` with the same classes should be fine.

File: static/js/src/public/details/integrations/components/App/App.tsx [hunk 2]
```diff
@@ -150,10 +150,15 @@ export const App = () => {
...
+                  <a href="https://juju.is/docs/juju/relation">
+                    Learn about integrations &gt;
```
Comment (steverydz) [inline static/js/src/public/details/integrations/components/App/App.tsx:159]:
```suggestion
                    Learn about integrations&nbsp;&gt;
```

**Review by steverydz — COMMENTED** [review]
One comment in the code and one suggestion

File: static/js/src/public/details/integrations/components/App/App.tsx (hunk 2 above)
Comment (chillkang) [reply static/js/src/public/details/integrations/components/App/App.tsx]: @steverydz I updated the changes!

**Review by steverydz — APPROVED** [review]
LGTM 👍

**Comment by lukasSerelis** [comment]
LGTM!

**Pull request merged**